src/irmin-graphql/unix/irmin_graphql_unix.mli

```diff
@@ -14,24 +14,27 @@
  * OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
  *)
 
+type remote_fn =
+  ?ctx:Mimic.ctx -> ?headers:Cohttp.Header.t -> string -> Irmin.remote Lwt.t
```

Comment:
I would like to have this function signature not contain `?ctx:Mimic.ctx` since it is not used but kept it for now to maintain focus of the changes.

Reply:
Hm, I wonder if it’s a bug that `ctx` is unused? I will try to look into that.

Reply:
:pray: It might be. I was planning to ask you about this if you didn't see the PR. Heads up that I'd also like to talk about `Resolver` (I copied this type from it since it was the only part of the dependency and to avoid a cyclic relationship) as I make subsequent changes.

Reply:
Ah okay, that makes sense. I’m pretty sure it’s used in resolver.ml to make the irmin-git remote functions work correctly. 

`Resolver` is pretty complicated so I’m happy to go over it sometime!